coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentControllerBase.java

```diff
@@ -255,11 +255,12 @@ public ModelAndView save(ProposalDevelopmentDocumentForm form) throws Exception
          ((ProposalDevelopmentViewHelperServiceImpl)form.getViewHelperService()).setOrdinalPosition(form.getDevelopmentProposal().getProposalPersons());
          saveAnswerHeaders(form, form.getPageId());
 
+         getTransactionalDocumentControllerService().save(form);
```

Comment:
Yeah, it is causing problem when audit rules are executed before save on document that we retrieved at the time of narrative save.